### ISSUE TITLE
Proposal - Allow new judges to appoint their own first judge mentor

### ISSUE THREAD

**Issue by 5z1punch** [code-423n4/org#120]
There is a new rule added to the Judge Guidelines, that "when judging an audit for the very first time, the new judge's work is reviewed by a senior judge.". This will be very helpful for new judges to get started with their work.

I hope to allow new judges to appoint their own first judge mentor. Allowing new judges to choose senior judges they trust and are more familiar with for assistance can lead to more efficient  collaboration.

**Comment by GalloDaSballo** [comment]
This was done in the past, where the new Judge would offer a portion of their Judging Fee to a Senior Judge to get advice and reviews

**Comment by liveactionllama** [comment]
Yes, just want to clarify that the requirement isn't new regarding a senior judge reviewing a new judge's work the first time. It has been consistently followed. I just noticed that it wasn't mentioned in our Judge Guidelines, so I recently added it there for more awareness, especially for new judges during onboarding.
